DataReleaseNotes.json

```diff
@@ -1,378 +1,395 @@
-          "list": ["110 Participants", "990 Files"]
-        },
-        {
-          "paragraph": "CCDI: Molecular Characterization Initiative – phs002790.v2.p1",
-          "list": ["108 Participants", "1,987 Files"]
-        }
-      ]
-    },
-    {
-      "id": "v2.0",
-      "versionNumber": "v2.0",
-      "releaseDate": "April 17, 2023",
-      "subHeading": "Studies",
-      "heading": "Data Release 2.0",
-      "content": [
-        {
-          "paragraph": "CCDI: UCSF Database for the advancement of JMML – Integration of metadata with “Omic” Data - phs002504.v1.p1",
-          "list": ["1,015 Participants", "14,291 Files"]
-        },
-        {
-          "paragraph": "Childhood Cancer Data Initiative (CCDI): Free the Data: Open Sharing of Comprehensive Genomic \nChildhood Cancer Datasets (Kansas) - phs002529.v1.p1",
-          "list": ["193 Participants", "5,374 Files"]
-        },
-        {
-          "paragraph": "DCCPS CIDR: Discovery, Biology, and Risk of Inherited Variants in Glioma sample – phs002250.v2.p1",
-          "list": ["146 Participants", "292 Files"]
-        },
-        {
-          "paragraph": "DCCPS:Discovery of Colorectal Cancer Susceptibility Genes in High-Risk Families – phs001787.v1.p1",
-          "list": ["51 Participants", "318 Files"]
-        },
-        {
-          "paragraph": "DCCPS: Detection of Colorectal Cancer Susceptibility Loci Using Genome-Wide Sequencing – phs001554.v1.p1",
-          "list": ["2,892 Participants", "5,784 Files"]
-        },
-        {
-          "paragraph": "HTAN: Human Tumor Atlas Network Primary Sequencing Data - phs002371.v2.p1 – phs002371.v3.p1",
-          "list": ["660 Participants", "3,781 Files"]
-        },
-        {
-          "paragraph": "PDXNet: University of Texas PDX Development and Trial Center Grant – phs001980.v2.p1",
-          "list": ["36 Participants", "98 Files"]
-        }
-      ]
-    },
-    {
-      "id": "v1.0",
-      "versionNumber": "v1.0",
-      "releaseDate": "February 22, 2023",
-      "subHeading": "Studies",
-      "heading": "Data Release 1.0",
-      "content": [
-        {
-          "paragraph": "DCCPS GECCO OICR: Molecular Pathological Epidemiology of Cancer - phs002050.v2.p",
-          "list": ["2,528 Participants", "5,056 Files"]
-        },
-        {
-          "paragraph": "HTAN: Human Tumor Atlas Network Primary Sequencing Data - phs002371.v2.p1",
-          "list": ["270 Participants", "14,808 Files"]
-        },
-        {
-          "paragraph": "CCDI: Molecular Characterization Initiative - phs002790.v1.p1",
-          "list": ["526 Participants", "9,763 Files"]
-        },
-        {
-          "paragraph": "CCDI: UCSF Database for the advancement of JMML – Integration of metadata with “Omic” Data - phs002504.v1.p1",
-          "list": ["188 Participants", "935 Files"]
-        }
-      ]
-    }
-  ]
-}
+    "HEADING": "Release Notes",
```

Comment:
Fix spacing, first space is 4 spaces, the rest are 2. Final bracket is also not aligned.